xdsl/dialects/scf.py

```diff
@@ -1,10 +1,18 @@
 from __future__ import annotations
+
 from typing import Annotated, List
 
-from xdsl.dialects.builtin import IntegerType
-from xdsl.ir import SSAValue, Block, Region, Operation, Attribute, Dialect
-from xdsl.irdl import (VarOpResult, VarOperand, irdl_op_definition, Operand,
-                       AnyAttr)
+from xdsl.dialects.builtin import IndexType, IntegerType
+from xdsl.ir import Attribute, Block, Dialect, Operation, Region, SSAValue
+from xdsl.irdl import (
+    AnyAttr,
+    Operand,
+    SingleBlockRegion,
+    VarOperand,
+    VarOpResult,
+    irdl_op_definition,
```

Comment:
nit:
```suggestion
    irdl_op_definition
```
Will be consistent with the rest of our imports